offset_lat: 41.50250101286 
offset_long: -81.6077728
lat_conversion_to_m: 111060.0
long_conversion_to_m: 83495.0

